src/frontends/tensorflow/src/input_model.hpp

```diff
@@ -16,11 +16,13 @@ namespace tensorflow {
 
 class OpPlace;
 class TensorPlace;
+class SMVariablesIndex;
 
 class InputModel : public ov::frontend::InputModel {
     friend class TranslateSession;
     class InputModelTFImpl;
     std::shared_ptr<InputModelTFImpl> _impl;
+    std::shared_ptr<SMVariablesIndex> m_variables_index;
```

Comment:
please hide this member inside `_impl` in cpp file

Reply:
I’ll take a look